app/index.js

```diff
@@ -81,6 +81,7 @@ module.exports = class GeneratorLicense extends Generator {
         name: 'license',
         message: 'Which license do you want to use?',
         default: this.options.defaultLicense,
+        when: this.options.license == null || licenses.find(x => x.value === this.options.license) === undefined,
```

Comment:
A comparison like that (`licenses.find(x => x.value === this.options.license) === undefined`) is never used in this generator. All the other conditions in the prompts are ok, unless I am missing something?